### PR TITLE
Bug/e UI 3621 collection search filter 2 aa

### PR DESCRIPTION
**Pull request by adnanakgun**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3621

### Change description ###

Introduced extra param to control collections inside the search block and fixed a bug

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by phillip-whitaker — DISMISSED** [review]
LGBTQ+

File: src/shared/components/palette/collection/write-collection-field.component.ts [hunk 1]
```diff
@@ -185,6 +183,10 @@ export class WriteCollectionFieldComponent extends AbstractFieldWriteComponent i
     }
   }
 
+  isSearchFilter(): boolean {
```
Comment (paul-graham) [inline src/shared/components/palette/collection/write-collection-field.component.ts:186]: _Nitpick:_
Should be `public`.

File: src/shared/components/palette/collection/write-collection-field.component.ts [hunk 2]
```diff
@@ -185,6 +183,10 @@ export class WriteCollectionFieldComponent extends AbstractFieldWriteComponent i
...
+    return this.isInSearchBlock && this.collItems.length > 0;
+  }
+
   addItem(doScroll: boolean): void {
```
Comment (paul-graham) [inline src/shared/components/palette/collection/write-collection-field.component.ts:190]: _Nitpick:_
Should be `public`.

File: src/shared/components/palette/base-field/abstract-field-write.component.ts [hunk 3]
```diff
@@ -11,6 +11,9 @@ export abstract class AbstractFieldWriteComponent extends AbstractFormFieldCompo
   @Input()
   isExpanded = false;
 
+  @Input()
+  isInSearchBlock = false;
```
Comment (paul-graham) [inline src/shared/components/palette/base-field/abstract-field-write.component.ts:15]: _Nitpick:_
Should be `public`.

**Review by paul-graham — APPROVED** [review]
Couple of nitpicks, but otherwise all looks to be in order. 🙂

**Review by phillip-whitaker — APPROVED** [review]
LGBTQ+

**Review by paul-graham — APPROVED** [review]

**Pull request merged**